lib/components/Insights/InsightsContainer/index.tsx

```diff
@@ -33,22 +33,24 @@ export const InsightsContainer = forwardRef<
 >(({ header, footer, children }, ref) => (
   <Card ref={ref}>
     <CardHeader>
-      <CardTitle>{header.title}</CardTitle>
       <CardDescription>{header.description}</CardDescription>
+      <CardTitle>{header.title}</CardTitle>
```

Comment:
Why the change of order?

Reply:
It's based on the designs at figma, but I will revert it since the design is not yet final.